core/model/transactions/transfer_transaction.cpp

```diff
@@ -3,25 +3,35 @@
 
 namespace transaction {
 
-    std::string TransferTransaction::getHash() {
-        return hash;
+    TransferTransaction::TransferTransaction(
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Member variable 'TransferTransaction::type' is not initialized in the constructor.](https://www.codacy.com/app/soramitsu/iroha/file/4274201891/issues/source?bid=3712501&fileBranchId=3837061#l6)